packages/utils/README.md

```diff
@@ -359,30 +359,21 @@ Tags after patch version are ignored, e.g. 1.0.0-beta.1 is considered equal to 1
 
 ### :wrench: Constants
 
-- [E8S_PER_TOKEN](#gear-e8s_per_token)
 - [ICPToken](#gear-icptoken)
 
-#### :gear: E8S_PER_TOKEN
-
-| Constant        | Type     |
-| --------------- | -------- |
-| `E8S_PER_TOKEN` | `bigint` |
```

Comment:
do we still need the constant in `constants.ts`?

Reply:
We can remove later. I'm trying to be backwards compatible now.

Reply:
We can remove. The constant is not use anymore and was not exposed.

Reply:
done